ocs_ci/utility/vsphere.py

```diff
@@ -542,6 +542,25 @@ def is_resource_pool_exist(self, pool, dc, cluster):
         """
         return True if self.get_pool(pool, dc, cluster) else False
 
+    def check_resource_pool_existence(self, pool_prefix, dc, cluster):
```

Comment:
can we rename to "is_resource_pool_prefix_exist" bcoz we have with similiar function name at  https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/vsphere.py#L530

Reply:
yes